ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/management-api-http.yml

```diff
@@ -0,0 +1,97 @@
+---
+management_api_http:
+  enable_http: false
+  enable_https: true
+  https_ssl_profile: SSL_PROFILE
+  default_services: true
+  enable_vrfs:
+    - name: default
+      access_group: ACL-API
+      ipv6_access_group: ACL-API6
+    - name: MGMT
+      access_group: ACL-API
+  protocol_https_certificate:
+    certificate: |-
+      -----BEGIN CERTIFICATE-----
+      MIIFNjCCAx4CCQCVGSFu9M4dNDANBgkqhkiG9w0BAQsFADBdMQswCQYDVQQGEwJD
+      QTELMAkGA1UECAwCQkMxEjAQBgNVBAcMCVZhbmNvdXZlcjEPMA0GA1UECgwGQXJp
+      c3RhMQwwCgYDVQQLDANBVkQxDjAMBgNVBAMMBWhvc3QxMB4XDTI0MTExMzE3NTAw
+      N1oXDTM0MTExMTE3NTAwN1owXTELMAkGA1UEBhMCQ0ExCzAJBgNVBAgMAkJDMRIw
+      EAYDVQQHDAlWYW5jb3V2ZXIxDzANBgNVBAoMBkFyaXN0YTEMMAoGA1UECwwDQVZE
+      MQ4wDAYDVQQDDAVob3N0MTCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCCAgoCggIB
+      AJz2eUVD/4u+nmz0Ak6QJqPrQSJ6spZtxnfgSD3ETGhWgVO+aV7UbnpCAvKB+G7B
+      X3WEHl2zphFE+O4HhRCBiDFB9LXZ2fiiBKcPmFMuJChL15wLaa5yP99trMX3wxSL
```

Comment:
what are these? Are they valid certs? What are they used for?

Reply:
@gmuloc 
Here is the thread
https://github.com/aristanetworks/avd/pull/4694#discussion_r1832721318

Initially it was invalid certificate which cause an issue if deploy via CVP. So Alexey helped to get the self-signed cert+key

Reply:
Hmm ok these will probably be flagged by secret-scanner so please add a comment in the file to indicate these are unused self-signed certificates used for test purposes only

Reply:
cf suggestion above

Reply:
Applied.